src/csharp/Microsoft.Spark.E2ETest/IpcTests/SparkContextTests.cs

```diff
@@ -21,11 +21,20 @@ public void TestSignaturesV2_3_X()
         {
             SparkContext sc = SparkContext.GetOrCreate(new SparkConf());
 
-            _ = sc.GetConf();
-            _ = sc.DefaultParallelism;
+            Assert.IsType<SparkConf>(sc.GetConf());
+            Assert.IsType<int>(sc.DefaultParallelism);
 
             sc.SetJobDescription("job description");
 
+            sc.SetLogLevel(SparkContext.LogLevel.ALL);

```

Comment:
actually, not sure if this experience is good. can we just use string?

Reply:
Sure, changed to use string